letsencrypt/views.py

```diff
@@ -28,10 +26,4 @@ def detail(request, acme_data):
         challenge=acme_data,
     )
 
-    return render(
-        request,
-        'letsencrypt/detail.html',
-        {
-            'response': acme_challenge.response
-        },
-    )
+    return HttpResponse(acme_challenge.response, content_type="text/plain")
```

Comment:
Looks like the build failed due to a missing `new line` at the end of the file:

```
./letsencrypt/views.py:29:76: W292 no newline at end of file
````

Instead of sending this back to you, I'll mark this review as changes requested, and I'm building another PR based off your commits (with your author information of course).